packages/api/__tests__/index.js

```diff
@@ -277,31 +278,56 @@ describe('/api/profile/createUser', () => {
       done();
     });
   });
-
-  it('should send a 201 response when a student is created', (done) => {
-    request({
-      url,
-      method: 'POST',
-      body: testUser,
-      json: true,
-      resolveWithFullResponse: true,
-    }).then((res) => {
-      expect(res.statusCode).toBe(201);
-      expect(res.body).toMatchObject({ user_created: 'Success', user_email: testUser.email });
-      done();
+  it('should return 201 on creation of new user', (done) => {
+    tokenService.generate({
+      email: 'vipulrawat007@gmail.com',
+      tokenType: 'EMAIL_VERIFICATION',
+    }).then((token) => {
+      request({
+        url: verifyTokenUrl,
+        method: 'POST',
+        body: { token },
+        json: true,
+        resolveWithFullResponse: true,
+      }).then((res) => {
+        const cooky = res.headers['set-cookie'][0].substr(6, 200);
```

Comment:
The hell is going on here?

Reply:
Check if the requests library has a better way to extract cookie values

Reply:
Cookie refers to the full key=val pair. Would be better to call this variable 'token'